knowledge-base/getting-telerik-reportViewer-is-not-a-function-console-error.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Getting telerik_reportViewer is not a function console error
+description: Upon running the report viewer, an error occurs in the browser console.
+type: troubleshooting
+page_title: Getting telerik_reportViewer is not a function console error
+slug: getting-telerik-reportViewer-is-not-a-function-console-error
+position: 
+tags: HTML5Viewer
+ticketid: 1410720
+res_type: kb
+---
+
+## Environment
+<table>
+    <tbody>
+	    <tr>
+	    	<td>Product</td>
+	    	<td>Progress® Telerik® Reporting</td>
+	    </tr>
+	    <tr>
+	    	<td>Project Type</td>
+	    	<td>All web-based viewers</td>
+	    </tr>
+    </tbody>
+</table>
+
+
+## Error Message
+```
+telerik_reportViewer is not a function
+```
+
+## Cause\Possible Cause(s)
+Page containing a HTML5 Viewer must load only once jQuery library of version greater than 1.9.1 (according to the [Kendo UI requirements](http://docs.kendoui.com/getting-started/javascript-dependencies#jquery-version)). For more information, please check [HTML5 Viewer Requirements](../html5-report-viewer-system-requirements) help article.
+
+Such error can occur if:
+	- jQuery is not loaded;
+	- jQuery is loaded more than once and with different version;
+	- jQuery is loaded in later time in the page life cycle.
+
+## Solution
+Check when the **head** section is loaded and if the link to jQuery v1.9.1 or greater is loaded before the viewer's script. 
```

Comment:
The head section of an HTML page always loads in the same order. Why should the users "check when the head section is loaded"?

Reply:
"The link to jQuery" is enough. You don't need to specify the version every single time. It is mentioned at the beginning of the article.